run-script.sh

```diff
@@ -1,59 +1,92 @@
 #!/usr/bin/bash -u
 
+curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /usr/local/bin/jq
+chmod +x /usr/local/bin/jq
+
 prefix=perf-test
+uuid=${UUID}
 num_users=${NUM_USERS:-30}
 rate=${RATE:-10}
 quay=${QUAY_URL}
 duration=${DURATION:-10}
 pick=$((1 + RANDOM % ${num_users}))
 
+prefix=${PREFIX:-perf-test}
+elastic=${ES:-search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com}
```

Comment:
(Same as above)
Do you want to keep this in your public Github repo?

Reply:
:) yeah, i don't care about this machine.